spritz/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">A Taste of Spritz</string>
```

Comment:
`cmd-alt-l` plz

Reply:
Done in ebe99aaeb3931424b1b1d4ae249401269a023e03.